src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -424,6 +444,7 @@ class LicenceController(
     )
   }
 
+  @Tag(name = Tags.LICENCE_CONDITIONS)
```

Comment:
Tags spread controllers - it might make more sense to restructure the controllers so operations are grouped into controllers via associated Tag